### DISCUSSION TITLE
Comand not found?

### DISCUSSION THREAD

**Discussion by Bandy325** [shannonhochkins/ha-component-kit#125] [Q&A] [answered]
First of all, thank you for all the effort you have already put into this. My question is that I have gone through the how-to twice but am still getting an error during installation. When I type the line "npx create-react-app dashboard --template typescript" in the terminal of studio code, I get the feedback "zsh comand not found: npx"? Given the feedback, I may have missed a "step" somewhere, I just can't figure out which one.

**Comment by jenseo** [comment]
Hey @Bandy325 ,
I'm not the author of this kit, but offering some advice at least :) It sounds like you're creating a new React App, which is perfectly fine, but to make it easier for you, there is a built in starter script that sets everything up for you automatically:

`npm create hakit@latest`

Have a look here if you haven't already:

https://shannonhochkins.github.io/ha-component-kit/?path=/docs/introduction-getting-started--docs

**Reply by Bandy325** [reply]
Thank you for the quick answer! I had indeed seen that too, unfortunately I get the same feedback in studio code. When I enter the line in SSH/Terminal I get the message bash: npm comand not found?

**Reply by jenseo** [reply, marked as answer]
Ah, sounds like you don't have npm package manager installed on your computer. Depending on if you're on Mac or Windows, there is a bit of a difference how you install it. You need to install nodejs and npm package manager, give it a run in Google and you'll find how to do that.

**Reply by Bandy325** [reply]
Ah, see, I knew it would be something so "stupid". As you can see, I'm not really into computers, but I enjoy learning. Thanks for your answer!

**Reply by shannonhochkins** [reply]
Thanks for helping out @jenseo ! I might need to uodat3 the docs to include prerequisites required to install!
